esileapclient/tests/unit/osc/v1/test_node.py

```diff
@@ -38,9 +38,44 @@ def test_node_list(self):
 
         collist = [
             "Name",
+            "Owner",
+            "Offer UUID",
+            "Lease UUID",
+            "Lessee"
         ]
 
         self.assertEqual(collist, list(columns))
 
-        datalist = ((fakes.node_name,),)
+        datalist = ((fakes.node_name,
+                     fakes.node_owner
+                     ),)
```

Comment:
I think you'll end up with empty strings for the additional fields here.

Reply:
Fixed